account_banking_pain_base/views/account_payment_method.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
```

Comment:
AFAIK we did never put copyrights in XML files. I think we should try to not be too nitpicky, for it can drag down motivation and contributions.